build_tools/azure/install.sh

```diff
@@ -24,6 +24,10 @@ make_conda() {
     source activate $VIRTUALENV
 }
 
+function version_ge() {
+    test "$(echo "$@" | tr " " "\n" | sort -rV | head -n 1)" == "$1";
```

Comment:
Maybe add a short comment on what this does? I don't really understand how it performs the greater or equal operation.